Modules/Loadable/VolumeRendering/MRML/CMakeLists.txt

```diff
@@ -4,7 +4,10 @@ set(KIT ${PROJECT_NAME})
 
 set(${KIT}_EXPORT_DIRECTIVE "VTK_SLICER_${MODULE_NAME_UPPER}_MODULE_MRML_EXPORT")
 
+find_package(JsonCpp REQUIRED)
```

Comment:
While a not show stopper, I am wondering if we should standardize on using RapidJson. It is already used in the terminology module and also in this PR. See https://github.com/Slicer/Slicer/pull/1017/files#diff-f636c178327b4ed7f9b21a1d8c5574de

Reply:
Can we leave it for the moment? Since both are used in Slicer at the moment, it shouldn't be a problem. I personally don't care which one is used (I made an arbitrary choice which I implemented this), but I don't want to rewrite this code now.

Reply:
> Can we leave it for the moment? 

Sounds good. Since it is only used internally, there is no rush. 

After adding the round-trip test for read/write, we will be able to switch later with confidence.